Feat data manager

alvin6233 opened the pull request:
评测方案管理列表&新建评测方案

Commit:
评测方案管理

Commit:
新建评测方案页面

Commit:
新建评测方案页面

Commit:
新建评测方案页面数据集表格添加及动态统计表格

Commit:
Merge branch 'dev' into feat-data-manager

# Conflicts:
#	dashboard/enums.py
#	dashboard/models.py
#	dashboard/resources.py
#	dashboard/routes.py

Commit:
新建评测方案页面数据集表格添加及动态统计表格

Commit:
新建评测方案页面数据集表格添加及动态统计表格

allenwoods commented on dashboard/biz_routers/datamanager.py:
```diff
@@ -0,0 +1,285 @@
+
+
+@router.get("/{resource}/epm_create")
+async def create_view(
```
此文件中的方法建议增加注释，包括所在页面、上下游依赖、输入输出示例，方便其它同学学习

alvin6233 replied:
👌🏻

allenwoods commented on dashboard/templates/evaluationplanmanager/copy_create.html:
`dashboard/templates/evaluationplanmanage`是不是太长了，`dashboard/templates/evaluation`？

Commit:
1. databoard/biz_router/datamanager.py添加注解描述相关接口
2. 重命名dashbord/models中Evaluation to EvaluationRecord，并调整相关依赖文件引用
3. 重命名dashbord/models中Evaluation to EvaluationPlanManager to EvaluationPlan, 并调整相关引用文件
4. 修改/dashboard/template/evaluationplan/update.html 修复页面方案名称及依赖数据集不显示的问题
